less/navigation.less

```diff
@@ -1 +1,44 @@
-// Navigation code here
\ No newline at end of file
+// Navigation code here
+
+ nav{
+     width: 70%;
```

Comment:
should use display flex on the nav with flex-direction column to stack the h1 and the links.  align-items center would handle centering the h1 and the links horizontally when using column.  would then not have to use margin.